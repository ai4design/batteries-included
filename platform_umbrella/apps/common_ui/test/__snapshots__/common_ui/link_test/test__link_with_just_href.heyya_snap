<a href="https://www.google.com/" class="">
  Test bare href
</a>