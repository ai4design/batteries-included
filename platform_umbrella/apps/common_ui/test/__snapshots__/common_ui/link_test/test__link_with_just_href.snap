"<a href=\"https://www.google.com/\" class=\"\">\n  Test bare href\n</a>"